docs/intelligentapps/promptbuilder.md

```diff
@@ -0,0 +1,66 @@
+---
+Order: 4
+Area: intelligentapps
+TOCTitle: Prompt Builder
+ContentId:
+PageTitle: Prompt Builder
+DateApproved:
+MetaDescription: Get Started with creating, iterating and optimizing your prompts in AI Toolkit.
+---
+# Prompt Engineering in AI Toolkit
+
+Prompt Builder in AI Toolkit streamlines the prompt engineering workflow by generating starter prompts, helping you iterate and refine with each run, breaking down complex tasks through prompt chaining and structured outputs, and providing easy access to code for seamless LLM integration via APIs.
+
+![PromptBuilder demo](./images/promptbuilder/promptbuilder.gif)
+
+## Create, edit and test prompts
+
+To access the Prompt Builder:
+- In AI Toolkit view, select **Prompt Builder**
+- Select **Try in Prompt Builder** from a model card in the model catalog
+
+To test a prompt in the prompt builder, follow these steps:
+1. In **Models**, select a model from the dropdown list or click **Browser models** to add another model from model catalog.
+    ![select_model](./images/promptbuilder/s1_models.png)
+2. Enter **System prompt** and **User prompt**
+   > [!TIP]
+   > - System prompt is optional, it is used to provide instructions with relevant context to guide the model response
+   > - User prompt is mandatory, it is the input you want to send to the model
+   > - If you don't know how to input these prompts, you can simply describe your project idea in natural language and let the AI-powered feature generate prompts for you to experiment with
+   > ![generate_prompts](./images/promptbuilder/generate_prompt.gif)
+3. Click **Run** to send the prompts to the selected model
+4. Optionally, you can click **Add Prompts** to add more User and Assistant prompts to the conversation, or **Use Response as Assistant Prompt** as the history and context you send to the model to further guide the model's behavior
+5. You can repeat the above steps to iterate over your prompts by observing the model response and making changes to the prompts
+
+## Structured output
+
+Structured output support helps you design prompts to deliver outputs in a structured, predictable format.
+![structured_output](./images/promptbuilder/structured_output.gif)
+
+To test a prompt in the prompt builder, follow these steps:
```

Comment:
Is prompt builder a brand or just a feature description? If not a brand, use lowercase everywhere.

Reply:
It's just a feature